test/dynamo/test_misc.py

```diff
@@ -7591,7 +7592,6 @@ def fn():
     @unittest.skipIf(sys.version_info >= (3, 13), "feature landed in 3.13")
     def test_get_instruction_source_311(self):
         def f():
-            # flake8: noqa
```

Comment:
@kit1980 I was going to fix this one by trying to disable something locally, but if I remove flake8: noqa for the whole file there are around 100 errors. I'd be happy to add in a few # noqa: specific errors to just ignore these, but it seems like a lot of shuffling code around. Which would you prefer I do: move the flake8: noqa to the top with a comment in this specific case, or add in specific # noqa: XXX around the file?